README.md

```diff
@@ -81,7 +81,7 @@ he has to say, and it largely aligns with our own experiences.
 
 The main differences are:
 
-1) Dave basically proposes doing away with the notion of a logging API in favor
+1. Dave basically proposes doing away with the notion of a logging API in favor
```

Comment:
This triggers the Markdown formatting of an ordered list which indents this paragraph.

The following paragraph isn't indented, however, which looks wrong to me. As I read it, both paragraphs are part of this first point. Some leading space characters might convince the parser/formatter. For example,

```md
1. first point, first paragraph

   first point, second paragraph

2. second point
```

1. first point, first paragraph

   first point, second paragraph

2. second point

Reply:
I saw no difference in formatting in the 2 editors I use (Goland and VSCode).  It also looks identical in github's rendering.  But, I've changed it back because it's not worth even typing out this comment.

Reply:
I used the [view file](https://github.com/go-logr/logr/blob/4d214da042a93669b17f065919dbbbd873fb30b9/README.md) link in the PR to see how it renders at GitHub.